sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2098,6 +2098,12 @@ object SQLConf {
         "defined by `from` and `to`.")
       .booleanConf
       .createWithDefault(false)
+
+  val LEGACY_ADD_DIRECTORY_USING_RECURSIVE = buildConf("spark.sql.legacy.addDirectory.recursive")
+    .doc(" When true, users can add directory by passing path of a directory to ADD FILE" +
+      "command of SQL. If false, then only a single file can be added.")
```

Comment:
nit: You need a single space in the head.

Reply:
Added a space in last line.

Reply:
@maropu Any more changes required for this one??